Readme.md

```diff
@@ -37,7 +37,7 @@ Here's a quick public tour of Open Library to get your familiar with the service
 
 ## Installation
 
-Run `docker-compose up` and visit http://localhost:8080
+Run `docker-compose up` and visit http://localhost:8080.
```

Comment:
Let's revert this one because it removes the ability to click on this URL.
```suggestion
Run `docker-compose up` and visit http://localhost:8080
```
The other changes all look good to me.